.github/workflows/linux-checks.yaml

```diff
@@ -77,3 +77,87 @@ jobs:
+  func-test-kuber:
+    name: func-test-kuber # don't change, this name is used in repository restricts!
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: /home/runner/work/
+      V2_PATH: /home/runner/work/assured-ledger/assured-ledger/ledger-core/v2
+      INSOLAR_IMAGE: ci:5000/assured-ledger
+      USE_MANIFESTS: ci
+      GOPROXY: https://proxy.golang.org,https://goproxy.io,direct
+      KUBECTL: kubectl --kubeconfig=/tmp/k3s.yaml
+      KUBECTL_VERSION: v1.18.2
+      KUBEVAL_VERSION: 0.15.0
+      K3S_TAG: v1.17.4-k3s1
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Set up go ${{env.GO_VERSION}}
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{env.GO_VERSION}}
+      - name: Add bin to path
+        run: echo "::add-path::${{env.GOPATH}}/bin"
+      - name: get kubectl
+        run: |
+          curl -L -s --create-dirs https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl -o bin/kubectl
+          chmod +x $GITHUB_WORKSPACE/bin/kubectl
+      - name: get kubeval
+        run: |
+          curl -L -s --create-dirs https://github.com/instrumenta/kubeval/releases/download/$KUBEVAL_VERSION/kubeval-linux-amd64.tar.gz | tar xvz --exclude='LICENSE' --exclude='README.md' -C $GITHUB_WORKSPACE/bin
+          chmod +x $GITHUB_WORKSPACE/bin/kubeval
+      - name: add workspace/bin to path
+        run: echo "::add-path::$GITHUB_WORKSPACE/bin"
+      - name: check resources
+        run: |
+          sudo lscpu
+          free -h
+      - name: fire up k3s
+        run: |
+          touch /tmp/k3s.yaml
+          docker run -d --name=k3s --net=host --privileged --tmpfs /run --tmpfs /var/run -e "K3S_TOKEN=197813248412582" -p 6443:6443 -v "$PWD"/deploy/kube/ci/registries.yaml:/etc/rancher/k3s/registries.yaml -v /tmp/k3s.yaml:/etc/rancher/k3s/k3s.yaml rancher/k3s:$K3S_TAG server
+        working-directory: ${{env.V2_PATH}}
+      - name: launch registry
+        run: docker run -d -p 5000:5000 --restart=always --name registry registry:2
+      - name: wait for k3s
+        run:  until ${{env.KUBECTL}} version; do echo "k3s is not up yet"; sleep 3; done
+      - name: describe node
+        run: |
+          ${{env.KUBECTL}} get no
+          ${{env.KUBECTL}} describe no
+          docker ps
+      - name: apply ingress
+        run:  ${{env.KUBECTL}} apply -f deploy/kube/ingress-nginx.yaml
+        working-directory: ${{env.V2_PATH}}
+      - name: build images
+        run: make docker-build
+        working-directory: ${{env.V2_PATH}}
+      - name: push assured ledger image to local registry
+        run: |
+          docker tag insolar/assured-ledger:latest localhost:5000/assured-ledger:latest
+          docker tag insolar/assured-ledger:latest ci:5000/assured-ledger:latest
+          docker push localhost:5000/assured-ledger:latest
```

Comment:
Something seems to be wrong. These steps are not listed in README. Either README is wrong or these steps are not needed.

Reply:
This is for k3s local registry, I'll add a comment